editor/translations/extract.py

```diff
@@ -100,7 +220,7 @@ def process_file(f, fname):
 print("Updating the editor.pot template...")
 
 for fname in matches:
-    with open(fname, "r") as f:
+    with open(fname, "r", encoding="utf8") as f:
```

Comment:
This will break Python 2 compatibility, you should use `open_utf8` from our `compat.py` (`import compat`).

Reply:
Ah nevermind, this is not part of the buildsystem so it doesn't matter.

But might be worth bumping the shebang to `python3` then.

Reply:
Ahh ok. Didn't see your msg. I will revert the changes.

Reply:
@akien-mga You want me to change to `#!/usr/bin/env python3`? I didn't understand the shebang part until now. 
The `/usr/bin/env` part I refer from this [discussion](https://stackoverflow.com/questions/22222473/shebang-doesnt-work-with-python3).

Reply:
Yeah that works fine.